compiler/il/ILOpCodeProperties.hpp

```diff
@@ -376,6 +488,118 @@
    /* .ifCompareOpCode      = */ TR::BadILOp,
    },
 
+   {
+   /* .opcode               = */ TR::irdbari,
+   /* .name                 = */ "irdbari",
+   /* .properties1          = */ ILProp1::LoadVar | ILProp1::Indirect | ILProp1::HasSymbolRef,
+   /* .properties2          = */ ILProp2::ValueNumberShare | ILProp2::SupportedForPRE| ILProp2::MayUseSystemStack | ILProp2::ReadBarrierLoad,
+   /* .properties3          = */ ILProp3::LikeUse,
+   /* .properties4          = */ 0,
+   /* .dataType             = */ TR::Int32,
+   /* .typeProperties       = */ ILTypeProp::Size_4 | ILTypeProp::Integer,
+   /* .childProperties      = */ ONE_CHILD(TR::Address),
+   /* .swapChildrenOpCode   = */ TR::BadILOp,
+   /* .reverseBranchOpCode  = */ TR::BadILOp,
+   /* .booleanCompareOpCode = */ TR::BadILOp,
+   /* .ifCompareOpCode      = */ TR::BadILOp,
+   },
+
+   {
+   /* .opcode               = */ TR::frdbari,
+   /* .name                 = */ "frdbari",
+   /* .properties1          = */ ILProp1::LoadVar | ILProp1::Indirect | ILProp1::HasSymbolRef,
+   /* .properties2          = */ ILProp2::ValueNumberShare | ILProp2::SupportedForPRE| ILProp2::MayUseSystemStack | ILProp2::ReadBarrierLoad,
+   /* .properties3          = */ ILProp3::LikeUse,
+   /* .properties4          = */ 0,
+   /* .dataType             = */ TR::Float,
+   /* .typeProperties       = */ ILTypeProp::Size_4 | ILTypeProp::Floating_Point,
+   /* .childProperties      = */ ONE_CHILD(TR::Address),
+   /* .swapChildrenOpCode   = */ TR::BadILOp,
+   /* .reverseBranchOpCode  = */ TR::BadILOp,
+   /* .booleanCompareOpCode = */ TR::BadILOp,
+   /* .ifCompareOpCode      = */ TR::BadILOp,
+   },
+
+   {
+   /* .opcode               = */ TR::drdbari,
+   /* .name                 = */ "drdbari",
+   /* .properties1          = */ ILProp1::LoadVar | ILProp1::Indirect | ILProp1::HasSymbolRef,
+   /* .properties2          = */ ILProp2::ValueNumberShare | ILProp2::SupportedForPRE| ILProp2::MayUseSystemStack | ILProp2::ReadBarrierLoad,
+   /* .properties3          = */ ILProp3::LikeUse,
+   /* .properties4          = */ 0,
+   /* .dataType             = */ TR::Double,
+   /* .typeProperties       = */ ILTypeProp::Size_8 | ILTypeProp::Floating_Point,
+   /* .childProperties      = */ ONE_CHILD(TR::Address),
```

Comment:
Shouldn't the child here be a double? Similarly for others. This is likely a copy/paste mistake. Can we fix all of them?